src/sample/StudentDetailImpl.java

```diff
@@ -1,50 +1,75 @@
-package sample;
+package seokhwan;
 
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
+import java.util.*;
 
 public class StudentDetailImpl implements StudentDetail {
     Map<Integer, Student> studentsMap = new HashMap<>();
 
     @Override
     public boolean addStudent(int id, Student student) {
-
+        if (studentsMap.containsKey(id)){
+            return false;
+        }
+        else {
+            studentsMap.put(id,student);
+            return true;
+        }
     }
 
     @Override
     public boolean removeStudent(int id) {
-
+        if (studentsMap.containsKey(id)){
+            studentsMap.remove(id);
+            return true;
+        }
+        else {
+            return false;
+        }
```

Comment:
```suggestion
        if (studentsMap.containsKey(id)) {
            studentsMap.remove(id);
            return true;
        } else {
            return false;
        }
```

만약 `else문`을 쓰는 경우에는 이렇게 괄호를 열고 닫는 형태가 더 가독성이 올라갑니다!